modules/spo-custom-location.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="spo-custom-location_{context}"]
+= Use a custom location for nested environments
```

Comment:
I'm sorry I didn't tell you earlier, but this option is not really needed (or supported) on OCP. It's more often used in nested environments like kind (IIRC?).

I know that the upstream docs are written in a confusing way. Sorry.

Can we remove this module?

Reply:
Easy fix :)